vocabularies/EntityRelationship.xml

```diff
@@ -0,0 +1,292 @@
+  </edmx:Reference>
+  <edmx:Reference Uri="https://sap.github.io/odata-vocabularies/vocabularies/Common.xml">
+    <edmx:Include Namespace="com.sap.vocabularies.Common.v1" Alias="Common" />
+  </edmx:Reference>
+
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="com.sap.vocabularies.EntityRelationship.v1" Alias="EntityRelationship">
+      <Annotation Term="Core.Description">
+        <String>Terms to better document the Entity Relationship model outside of the current API / data model context</String>
+      </Annotation>
+      <Annotation Term="Core.Description" Qualifier="Published">
+        <String>2020-03-03 © Copyright 2020 SAP SE. All rights reserved</String>
+      </Annotation>
+      <Annotation Term="Core.Links">
+        <Collection>
+          <Record>
+            <PropertyValue Property="rel" String="latest-version" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/EntityRelationship.xml" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="alternate" />
+            <PropertyValue Property="href" String="https://sap.github.io/odata-vocabularies/vocabularies/EntityRelationship.json" />
+          </Record>
+          <Record>
+            <PropertyValue Property="rel" String="describedby" />
+            <PropertyValue Property="href" String="https://github.com/sap/odata-vocabularies/blob/main/vocabularies/EntityRelationship.md" />
+          </Record>
+        </Collection>
+      </Annotation>
+
+      <Term Name="entityType" Type="Edm.String" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" >
+          <String>
+            Defines which Entity Type the current JSON Schema object represents. 
+            There could be several JSON Schema objects that are assigned to the same Entity Type. 
+            One JSON Schema object can only have one Entity Type assigned, which corresponds to the 
+            applications own Entity Type definition. The ID scheme for an Entity Type ID is as following: 
+            entityTypeId := namespace:entityTypeLocalId[:v"majorVersion"]
+          </String>
+        </Annotation>
+      </Term>
+
+      <Term Name="propertyType" Type="Edm.String" Nullable="false" AppliesTo="Property">
+        <Annotation Term="Common.Experimental" />      
+        <Annotation Term="Core.Description" String="logical property type of a property" />
+        <Annotation Term="Core.LongDescription">
+          <String>Defines the logical Property Type of a property.
+            The reason is to have an ID to relate to the property type independently of the property name and shared 
+            across every usage of the same property across APIs where names might be different. 
+            Especially to state that it can be used as an ID or is part of a composite ID.
```

Comment:
```suggestion
            Especially to state that it can be used as an ID or is part of a composite ID.
            A Property Type ID follows the same format and considerations as the Entity Type ID explained above:
            `<PropertyTypeId> := <namespace>:<propertyTypeLocalId>[:v<majorVersion>]`
```

Reply:
+1, except the <> doesn't work